src/components/ProfileUI/ProfileUI.jsx

```diff
@@ -1,16 +1,21 @@
 import React from "react";
 
-const ProfileUI = ({ name, surname }) => (
-  <div>
-    name: {name}
-    <br />
-    surname: {surname}
+const ProfileUI = ({ name, surname, profile_pic }) => (
+  <div className="div_profile-view">
+    <div className="div__profile-view-background">
+      <div className="div__background-image"></div>
+      <img src={profile_pic} className="img__profile-view" />
+      <h2 className="h2__profile-view">
+        {name} {surname}
+      </h2>
+    </div>
```

Comment:
Correct class names heare are:

```
profile-view
profile-view__background
profile-view__image
profile-view__title
```

Reply:
I seem to have been mistaken about BEM. Thank you for the examples! Fixed it